src/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="icon" type="image/x-icon" href="./assets/images/favicon.ico" />
```

Comment:
As you have a favicon injected by webpack.
It would be better if you remove this one from your template:
https://github.com/muneebulrehman/todo-review/blob/582e368b6867ed8f4e7683e7bd3ffe52f964ef53/src/index.html#L7

Reply:
Fixed it. Thanks 😊